src/test/java/org/jboss/xavier/analytics/rules/BaseIntegrationTest.java

```diff
@@ -43,4 +47,13 @@ public void setup()
     public void tearDown()
```

Comment:
I see it's not part of this PR, but why this empty method ?

Reply:
Removed :+1: